intellij-plugin/README.md

```diff
@@ -0,0 +1,24 @@
+# Arcs IntelliJ Plugin
+
+An IntelliJ plugin supporting Arcs manifests (*.arcs files).
+
+## For Users
+
+1) Install the plugin from build/distributions/arcs_gradle_plugin-1.0.zip
```

Comment:
What commands do users have to run to build this zip file? Presumably they need to install gradle and then run the :buildPlugin command? Worth adding here?

Reply:
You don't need to install gradle: it's bundled into this project.

Reply:
There is no `build/distributions/arcs_gradle_plugin-1.0.zip` checked in, how does the user build it? There is only one step listed but surely there must be another.